homeassistant/components/onvif/camera.py

```diff
@@ -95,6 +110,10 @@
         pan = service.data.get(ATTR_PAN, None)
         tilt = service.data.get(ATTR_TILT, None)
         zoom = service.data.get(ATTR_ZOOM, None)
+        distance = service.data.get(ATTR_DISTANCE, None)
```

Comment:
Please use `dict[key]` for required schema keys and keys with default schema values.

Reply:
I don't figure out what that mean yet, will try to check against other components

Reply:
Like this:
```suggestion
        distance = service.data[ATTR_DISTANCE]
```

Reply:
Thank you